cpp/CMakeLists.txt

```diff
@@ -230,3 +230,17 @@ target_compile_options(example_auto_serialize PUBLIC ${SANITIZER_COMPILE_OPTIONS
 target_include_directories(example_auto_serialize SYSTEM PRIVATE ${json_SOURCE_DIR}/include)
 target_link_options(example_auto_serialize PUBLIC ${SANITIZER_LINK_OPTIONS})
 target_link_libraries(example_auto_serialize foxglove_cpp_static)
+
+option(BUILD_OPENCV_EXAMPLE "Build the RGB camera visualization example (requires OpenCV)" OFF)
+
+if(BUILD_OPENCV_EXAMPLE)
```

Comment:
could we use [EXCLUDE_FROM_ALL](https://cmake.org/cmake/help/v3.31/prop_tgt/EXCLUDE_FROM_ALL.html) for this?

Reply:
I added EXCLUDE_FROM_ALL for this executable. Let me know if you had something else in mind